src/client/controller/CurvytronController.js

```diff
@@ -93,6 +93,8 @@ CurvytronController.prototype.onConnect = function(e)
 {
     this.$scope.status = 'online';
     this.applyScope();
+
+    this.client.addEvent('dispatchNbPlayers');
```

Comment:
The event has to be requested by the client, because we need to have initialized the connection.